CHANGELOG.md

```diff
@@ -5,25 +5,56 @@ All notable changes to Zebra are documented in this file.
   This means `zebrad` no longer contains users' viewing keys.
-- We're no longer using general conditional compilation attributes for `tor`,
-  but only feature flags instead.
-- Fixed a bug with trailing characters in the openapi spec method descriptions ([#8597](https://github.com/ZcashFoundation/zebra/pull/8597))
-- Added default constructions for several RPC method responses ([#8616](https://github.com/ZcashFoundation/zebra/pull/8616))
-- Added Windows to the list of supported platforms in Tier 2 ([#8637](https://github.com/ZcashFoundation/zebra/pull/8637))
-- Zebra now drops transactions with unpaid actions in block templates and from the mempool.
+- Support for custom Testnets and Regtest is greatly enhanced.
+- Windows is now back in the second tier of supported platforms.
+- The end-of-support time interval is set to match `zcashd`'s 16 weeks.
+- The RPC serialization of empty treestates matches `zcashd`. 
 
 ### Added
 
-- Added a page to the Zebra book describing custom Testnets ([#8636](https://github.com/ZcashFoundation/zebra/pull/8636))
+- Add an init function for a standalone `ReadStateService` ([#8595](https://github.com/ZcashFoundation/zebra/pull/8595))
+- Add new parameters for custom Testnets ([#8477](https://github.com/ZcashFoundation/zebra/pull/8477), [#8518](https://github.com/ZcashFoundation/zebra/pull/8518), [#8524](https://github.com/ZcashFoundation/zebra/pull/8524))
+- Add default constructions for several RPC method responses ([#8616](https://github.com/ZcashFoundation/zebra/pull/8616))
+- Allow custom Testnets to make peer connections and configure more parameters ([#8528](https://github.com/ZcashFoundation/zebra/pull/8528))
+- Allow configurable NU5 activation height on Regtest ([#8505](https://github.com/ZcashFoundation/zebra/pull/8505))
+
+#### Docs
+
+- Add a section on Regtest usage to the Zebra book ([#8526](https://github.com/ZcashFoundation/zebra/pull/8526))
+- Describe custom Testnets in the Zebra book ([#8636](https://github.com/ZcashFoundation/zebra/pull/8636))
+
+### Changed
+
+- Drop transactions with unpaid actions ([#8638](https://github.com/ZcashFoundation/zebra/pull/8638))
+- Lower the mandatory checkpoint height ([#8629](https://github.com/ZcashFoundation/zebra/pull/8629))
+- Use the new `zcash_script` callback API ([#8566](https://github.com/ZcashFoundation/zebra/pull/8566))
+- Put Windows in Tier 2 of supported platforms ([#8637](https://github.com/ZcashFoundation/zebra/pull/8637))
+- Remove support for starting the scanner at `zebrad` startup ([#8594](https://github.com/ZcashFoundation/zebra/pull/8594))
+- Reduce the end of support time from 20 weeks to 16 weeks ([#8530](https://github.com/ZcashFoundation/zebra/pull/8530))
+- Restore parts of the internal-miner feature for use on Regtest ([#8506](https://github.com/ZcashFoundation/zebra/pull/8506))
+
+### Fixed
+
+- Allow square brackets in network config listen addr and external addr ([#8504](https://github.com/ZcashFoundation/zebra/pull/8504))
```

Comment:
Optional/Nitpick: This was working before as well but required users to explicitly define a port in their `listen_addr`

```suggestion
- Allow square brackets in network config listen addr and external addr without explicitly configuring a port in the listen addr ([#8504](https://github.com/ZcashFoundation/zebra/pull/8504))
```

Side note: I'm not sure if it's actually correct to define an IP address with the brackets and without a port, but, it's clear what the users mean. I'd want to log a warning when this happens in case the same structure doesn't work elsewhere (like in the RPC `listen_addr`), but I've had some difficulty using the `tracing` crate in the `serde` `Deserialization` impl, does anyone know why that doesn't seem to work?